corehq/apps/users/views/__init__.py

```diff
@@ -1004,3 +1004,12 @@ def location_restriction_for_users(request, domain):
         project.location_restriction_for_users = json.loads(request.POST["restrict_users"])
     project.save()
     return HttpResponse()
+
+
+@require_POST
+@require_superuser
+def register_fcm_device_token(request, couch_user_id, device_token):
+    user = WebUser.get_by_user_id(couch_user_id)
+    user.set_fcm_device_token(device_token)
```

Comment:
I don't think you need an explicit method here - you can just set the field itself